.travis.yml

```diff
@@ -5,7 +5,8 @@ sudo: false
 go:
   - "1.9"
```

Comment:
When updating supported versions on Travis we tend to drop support for the second oldest version (i.e 1.9). 
```suggestion
```